perf-tests/src/main/scala/sttp/tapir/perf/Common.scala

```diff
@@ -1,19 +1,15 @@
 package sttp.tapir.perf
 
-import sttp.tapir.{PublicEndpoint, endpoint, path, stringBody}
-
-import scala.io.StdIn
+import java.io.File
+import java.nio.file.Path
+import java.util.Date
+import scala.util.Random
 
 object Common {
-  def genTapirEndpoint(n: Int): PublicEndpoint[Int, String, String, Any] = endpoint.get
-    .in("path" + n.toString)
-    .in(path[Int]("id"))
-    .errorOut(stringBody)
-    .out(stringBody)
+  val rootPackage = "sttp.tapir.perf"
+  val LargeInputSize = 5 * 1024 * 1024
+  val Port = 8080
+  val TmpDir: File = new java.io.File(System.getProperty("java.io.tmpdir")).getAbsoluteFile
+  def tempFilePath(): Path = TmpDir.toPath.resolve(s"tapir-${new Date().getTime}-${Random.nextLong()}")
```

Comment:
that's a path to a *new* temp file?

Reply:
Yes, I'll rename this function to make it clearer.